server/icons/BrandIfpi.tsx

```diff
@@ -0,0 +1,28 @@
+export default function IconBrandIfpi({
+	size = 24,
+	color = 'currentColor',
+	stroke = 2.,
+	...props
+}) {
+	return (
+		<svg
+			xmlns='http://www.w3.org/2000/svg'
+			class='icon icon-tabler icon-tabler-brand-ifpi'
+			width={size}
+			height={size}
+			viewBox='0 0 24 24'
+			stroke-width={stroke}
+			stroke={color}
+			fill='none'
+			{...props}
+		>
+			<g transform='translate(-343.3 -311.3)' stroke-linecap='butt' stroke-linejoin='butt'>
```

Comment:
```suggestion
			<g transform='translate(-343.3 -311.3)' stroke-linecap='butt' stroke-linejoin='miter'>
```
The old value is not valid for stroke-linejoin, only for linecap. Setting this to the default value will fix the TS error and causes no visible difference.
But there is a general issue with `icon-sprite.svg.tsx` which I will address after merging, you shouldn't be forced to set these default values on the `g` element.